entry_types/scrolled/package/src/frontend/VideoPlayer.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import {useFile} from '../entryState';
+import {MediaPlayer} from './MediaPlayer';
+
+
+/**
+ * Render video file in MediaPlayer.
+ *
+ * @param {Object} props
+ * @param {number} props.id - Perma id of the video file.
+ */
+export function VideoPlayer(props) {
+  const videoFile = useFile({collectionName: 'videoFiles', permaId: props.id});
+  const processSources = file => [
+    {type: 'video/ogg', src: `${file.urls.ogg}?u=1`},
+    {type: 'video/mp4', src: `${file.urls.m4a}?u=1`},
+    {type: 'video/mp3', src: `${file.urls.mp3}?u=1`}
+  ];
```

Comment:
These sources do not make sense. You can look at the old implementation as a reference ([1], [2]). Since we won't be doing bandwidth detection, you can ignore everything regarding that and always use high quality.

[1] https://github.com/codevise/pageflow/blob/master/entry_types/paged/packages/pageflow-paged-react/src/media/components/VideoFilePlayer/sources.js
[2] https://github.com/codevise/pageflow/blob/master/entry_types/paged/packages/pageflow-paged-react/src/media/components/VideoFilePlayer/__spec__/sources-spec.js

Reply:
could you please write how sources should look like? 
it is difficult to decode from the mentioned references

Reply:
I'd say it's basically in the test:

* All files have mp4 and hls urls
* Some files have Dash urls (Dash was only added in a later version, so files encoded before do not have these outputs
* If the user requested a specific quality (medium, fullhd) include only that url in the sources
* Fullhd or 4K are only available of the originally uploaded video has at least that quality. So even if the user requested 4K, we have to fallback to a lower quality if the requests quality is not present.

Reply:
If I understand correctly in current state it should just use high quality url file.urls['high'] ? 
or should I implement that sources function separately ?

Reply:
We need the sources logic eventually. We could call this PR done without it, but since it is blocked by the jQuery stuff anyway, we could also continue here.